sdks/java/testing/nexmark/src/main/java/org/apache/beam/sdk/nexmark/queries/sql/SqlQuery2.java

```diff
@@ -51,18 +51,18 @@
   private final long skipFactor;
   private final Class<? extends QueryPlanner> plannerClass;
 
-  private SqlQuery2(String name, long skipFactor, Class<? extends QueryPlanner> plannerClass) {
+  private SqlQuery2(long skipFactor, Class<? extends QueryPlanner> plannerClass) {
     super("SqlQuery2");
```

Comment:
```suggestion
    super(name);
```

I think this name should be plumbed through here instead